src/pages/contact.tsx

```diff
@@ -41,18 +45,19 @@ const Contact = () => {
       <Header pageTitle='Contact' />
       <TopNav />
       <main className='h-[calc(100%-7rem)]'>
-        <div className='flex h-full flex-col p-4'>
-          {/* <div className='mx-auto '> */}
-          {getDisplayContactIndex() === -1 && !getCreatingNewContact() && (
-            <div className='m-auto flex h-14 w-full flex-row'>
-              <div className='flex-1'></div>
-              <h1 className='m-3 flex-1 text-center text-2xl'>Contacts</h1>
-              <div className='m-auto flex-1 text-center'>
-                <Button onClick={() => setCreatingNewContact(true)}>Add</Button>
-              </div>
+        {getDisplayContactIndex() === null && !getCreatingNewContact() && (
+          <div className='m-auto flex h-14 max-w-md flex-row'>
+            <div className='flex-1'></div>
+            <h1 className='m-3 flex-1 text-center text-2xl'>Contacts</h1>
+            <div className='m-auto flex-1 text-center'>
+              <Button size='medium' onClick={() => setCreatingNewContact(true)}>
+                Add
+              </Button>
             </div>
-          )}
-          {getDisplayContactIndex() === -1 && !getCreatingNewContact() && (
+          </div>
+        )}
+        <div className='m-auto max-w-md'>
+          {getDisplayContactIndex() === null && !getCreatingNewContact() && (
             <div className='border-grey m-2 flex flex-row rounded-xl border-2'>
```

Comment:
maybe call the getters and store the values in some constants before the jsx to reduce the number of function calls